src/main/resources/application-dev.properties

```diff
@@ -12,6 +12,12 @@ ccd.data.store.api.url=http://localhost:4452
 
 tornado.url=http://localhost:9095/api/render
 case_document_am.url =http://localhost:4455
-feature.secure-doc-store.enabled = ${SECURE_DOC_STORE_FEATURE:false}
+feature.secure-doc-store.enabled = true
 
-queue.create-updates.queue-name = ${CREATE_UPDATES_QUEUE_NAME}
\ No newline at end of file
+
+queue.create-updates.send.connection-string = Endpoint=sb://ethos-servicebus-devs.servicebus.windows.net/;SharedAccessKeyName=SendSharedAccessKey;SharedAccessKey=JpyFNVxQTESoFxCdwXtZXbz2MUT1Zlstdx7RLJL/0AA=;EntityPath=create-updates-tb
```

Comment:
This is your local dev settings. It shouldn't be in the commit. You don't need to change these values - use environment variables